cmd/podman/pod_inspect.go

```diff
@@ -44,6 +44,11 @@ func podInspectCmd(c *cliconfig.PodInspectValues) error {
 		pod *adapter.Pod
```

Comment:
I can't comment up higher, but the message at line 37 says:  "Act on the latest container podman is aware of".  I think that should be "latest pod podman".